service/history/nDCBranchMgr.go

```diff
@@ -204,6 +204,7 @@ func (r *nDCBranchMgrImpl) verifyEventsOrder(
 	if incomingFirstEventID > nextEventID {
 		executionInfo := r.mutableState.GetExecutionInfo()
 		return false, newNDCRetryTaskErrorWithHint(
+			"Resend events due to out of order delivery",
```

Comment:
nit: Also create constants for the string? Same for others.

Reply:
Updated